### PR TITLE
Updated reponse.js (Conclusion)

### LINKED ISSUE #172: Draft conclusion page

### PR DESCRIPTION
**Pull request by satwikgoyal**
Used the QA component and added place holder body and questions field.

### CODE REVIEW

**Review by rasdy — CHANGES_REQUESTED** [review]
Looks good, but we should renamed the whole response page to "conclusion" (file name, component name, title, etc.)

File: frontend/src/pages/finalReflection.js [hunk 1]
```diff
@@ -45,17 +45,17 @@ function FinalReflection({ pages, setPages, activePage, setActivePage }) {
     setActivePage((prevPage) => "feedback");
   }
 
-  function goToResponse() {
-    if (pages.response.completed) {
-      if (pages.response.completed) {
-        if (!pages.response.visited) {
+  function goToConclusion() {
+    if (pages.conclusion.completed) {
+      if (pages.conclusion.completed) {
```
Comment (rasdy) [inline frontend/src/pages/finalReflection.js]: Double `if`'s?

File: frontend/src/pages/conclusion.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        </Grid>
+        <Grid item style={{ marginRight: "0rem", marginTop: "-3rem" }}>
+          {/*<Button variant="outlined">Next</Button>*/}
+        </Grid>
+      </Grid>
+      <Grid container spacing={2}>
+        <Grid item lg={12}>
+          <Box m="2rem">
+          </Box>
+          <QA header={bodyText} questions={questions} pages={pages} nextPageName={"conclusion"} />
```
Comment (rasdy) [inline frontend/src/pages/conclusion.js]: Hitting `submit` results in an exception

**Review by rasdy — CHANGES_REQUESTED** [review]
I noticed two small issues

File: frontend/src/pages/finalReflection.js (hunk 1 above)
Comment (satwikgoyal) [reply frontend/src/pages/finalReflection.js]: The double ifs were already there but you are right, I should have noticed this. Thanks!

File: frontend/src/pages/conclusion.js (hunk 2 above)
Comment (satwikgoyal) [reply frontend/src/pages/conclusion.js]: Resolved in the latest commit. Thanks a lot for the reviews and sorry about the minor mistakes.

**Review by rasdy — APPROVED** [review]
👍

**Pull request merged**